packages/kit/src/views/Home/pages/HomePageView.tsx

```
@@ -323,19 +349,20 @@ export function HomePageView({
               pt="$5"
               width="100%"
+              onLayout={handleTabPageLayout}
             >
               <TabPageHeader sceneName={sceneName} tabRoute={ETabRoutes.Home} />
             </YStack>
           ) : null}
         </Page.Body>
       </>
     );
-  }, [ready, wallet, sceneName, renderHomePageContent]);
+  }, [ready, wallet, sceneName, handleTabPageLayout, renderHomePageContent]);
```

ESLint

[error] React Hook useCallback has a missing dependency: 'tabPageHeight'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)